presto-hive/src/test/java/com/facebook/presto/hive/TestShowStats.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive;
+
+import com.facebook.presto.tests.AbstractTestQueryFramework;
+import com.google.common.collect.ImmutableList;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.hive.HiveQueryRunner.createQueryRunner;
+
+public class TestShowStats
+        extends AbstractTestQueryFramework
+{
+    public TestShowStats()
+    {
+        super(() -> createQueryRunner(ImmutableList.of()));
+    }
+
+    @Test
+    public void testShowStats()
```

Comment:
can you add some negative tests?

Reply:
@kokosing Negative tests already exist in `com.facebook.presto.tests.AbstractTestQueries#testShowStatsWith***Fails`

Reply:
I would move them here.